src/public/main.js

```diff
@@ -6,9 +6,34 @@ import { render } from 'react-dom'
 document.addEventListener('DOMContentLoaded', init)
 
 class App extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = []
+  }
+
+  addAll (photos) {
+    this.setState(state => state.concat(photos))
+  }
+
+  componentDidMount () {
+    const album = {
+      _type: 'io.cozy.photos.albums',
+      _id: 'f6f96990b5af9be2b04ca9f832000c5f'
```

Comment:
This code is a hard-coded ID for development purpose, as soon as I'll work on https://trello.com/c/hhjVyIhm/281-2-pho-29-partage-d-un-album-par-lien-liste-des-photos-de-l-album-partage-dans-la-page-publique it will be replaced by something more useful.